v2/auth/secretsmanagerauth/sign_in.go

```diff
@@ -99,7 +100,11 @@ func SignIn(ctx context.Context) (err error) {
 func signIn(ctx context.Context) (tokens auth.Tokens, err error) {
 	svc := secretsmanager.New(config.AWSSession)
 
-	secretKey := "arn:aws:secretsmanager:" + config.AWSSecretsManagerRegion + ":" + config.AWSSecretsManagerAccount + ":secret:" + config.SecretKey
+	var secretKey string
+
+	if secretKey = config.SecretKeyARN; secretKey == "" {
```

Comment:
I think somethine like this reads a bit easier :)
```suggestion
       secretKey := config.SecretKeyARN
       if secretKey == "" {
```